src/components/Clock.tsx

```diff
@@ -0,0 +1,62 @@
+import { Component } from 'react';
+
+type State = {
+  currentTime: Date,
+};
+
+type Props = {
+  clockName: string,
+};
+
+export class Clock extends Component<Props, State> {
+  state = {
+    currentTime: new Date(),
+  };
+
+  timerId = 0;
+
+  componentDidMount(): void {
+    this.timerId = window.setInterval(() => {
+      this.updateClock();
+
+      // eslint-disable-next-line no-console
+      console.info(this.state.currentTime.toUTCString().slice(-12, -4));
```

Comment:
U can create function, and give argument (date: Date) => date.toUTCString().slice(-12, -4); , than use in console info